wrt/wrt-uxmanu-android-tests/wrt-ux-app/testscripts/Crosswalk_WebApp_HTML5_Notification.feature

```diff
@@ -0,0 +1,7 @@
+Feature: wrt-ux-app
+ Scenario: Crosswalk WebApp HTML5 Notification
+  When launch "web_feature_notification_tests"
+    And I click button "Get Notification"
+    And I open notification
+   Then I should see text "New Email Received"
```

Comment:
The notification will be closed after press home hardware key,  so I add the last step.
And I will move it to be the last one running in tests.xml.